api/v1beta1/slicegateway_types.go

```diff
@@ -53,6 +53,8 @@ type SliceGatewayConfig struct {
 	SliceGatewayNodePort int `json:"sliceGatewayNodePort,omitempty"`
 	// Remote Node IP
 	SliceGatewayRemoteNodeIP string `json:"sliceGatewayRemoteNodeIp,omitempty"`
+	// Remote Node IPs
```

Comment:
Can we remove or deprecate the field `SliceGatewayRemoteNodeIP` ?

Reply:
removed